src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -4929,6 +4929,10 @@ namespace Akka.Util
         public override bool IsRight { get; }
         public TB Value { get; }
     }
+    public class static RuntimeDetector
```

Comment:
New class used to detect runtimes. Decided to unify all of our Mono detector implementations into a single static utility class.